tests/test_dump.py

```diff
@@ -30,12 +32,14 @@ def test_dump_load(tmpfile, index_data):
     index.dump()
     index.close()
 
+    os.system('rm {}'.format(os.path.join(tmpfile, 'metas.db')))
```

Comment:
we should refactor the `dump()` function, rather than just ignore this error in unittest.

Reply:
How could you think it is acceptable that the user must manually delete the meta data file at the end of the script in the real world? It does not make any sense.